tests/integration/targets/cli_parse/tasks/centos_native.yaml

```diff
@@ -5,3 +5,17 @@
     parser:
       name: ansible.netcommon.native
     set_fact: myfact
+
+- name: "{{ parser }} Run command which output nothing and parse with native"
+  ansible.utils.cli_parse:
+    command: "touch test_output"
+    parser:
+      name: ansible.netcommon.native
+    ignore_errors: true
```

Comment:
Why are we specifying ignore_errors here?

Reply:
Ack